generated/nifake/nifake/unit_tests/test_library_interpreter.py

```diff
@@ -410,6 +409,49 @@ def test_read_with_warning(self):
             assert test_error_desc in str(w[0].message)
             assert f'Warning {test_error_code} occurred.' in str(w[0].message)
 
+    def test_library_singleton_get_returns_same_instance(self):
+        # Test that if library singleton get is called multiple times it returns the same instance
+        interpreter = nifake._library_interpreter.LibraryInterpreter('windows-1251')
```

Comment:
I think this test will be more readable if you

1. create two LibraryInterpreter objects
2. assert that the LibraryInterpreter objects are not the same.
3. assert that the _library objects are the same instance